Space Navigator Unity project/Assets/SpaceNavigator/Plugins/SpaceNavigatorWindows.cs

```diff
@@ -11,29 +11,39 @@ class SpaceNavigatorWindows : SpaceNavigator {
 		private const float TransSensScale = 0.0001f, RotSensScale = 0.0008f;
 
 		// Public API
-		public override Vector3 GetTranslation() {
+		public override Vector3 GetTranslation()
+		{
+		    if (SubInstance._sensor != null && Translation == null)
+		        Translation = SubInstance._sensor.Translation;
+
 			float sensitivity = Application.isPlaying ? Settings.PlayTransSens : Settings.TransSens[Settings.CurrentGear];
 			return (SubInstance._sensor == null ?
```

Comment:
Testing `SubInstance._sensor == null` twice.  Can be put at the top of the method with an early return.